### PR TITLE
Docs: added an accessibility section in the pull request guide

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
As part of our ongoing work with accessibility in Grafana, we feel there is a need for us to add a section to the pull request guide to enable contributors to create UI components with accessibility in mind.
**Which issue(s) this PR fixes**:

Relates to #35613

### CODE REVIEW

File: contribute/create-pull-request.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ Pull requests that add or modify unit tests that are written in Jest must adhere
 - Don't add snapshots tests. We are incrementally removing existing snapshot tests, we don't want more.
 - If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), unless you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.
 
+Pull requests that creates new UI components must adhere to these accessibility guidelines:
+
+- Use semantic HTML.
+- Avoid using aria roles, labels and other accessibility attributes incorrectly. Accessibility attributes should only be used when semantic HTML doesn't satisfy your use case.
+- Use the Grafana theme palette for styling; contains colors with good contrast which aids accessibility.
```
Comment (marcusolsson) [inline contribute/create-pull-request.md:55]:
```suggestion
- Use the Grafana theme palette for styling. It contains colors with good contrast which aids accessibility.
```

We should also point the user to where can they learn how to use the theme palette.

File: contribute/create-pull-request.md [hunk 2]
```diff
@@ -48,6 +48,13 @@ Pull requests that add or modify unit tests that are written in Jest must adhere
...
+- Use [RTL](https://testing-library.com/docs/dom-testing-library/api-accessibility/) for writing unit tests; helps a lot in creating accessible components.
```
Comment (marcusolsson) [inline contribute/create-pull-request.md:56]:
```suggestion
- Use [RTL](https://testing-library.com/docs/dom-testing-library/api-accessibility/) for writing unit tests. It helps to create accessible components.
```

File: contribute/create-pull-request.md (hunk 1 above)
Comment (thisisobate) [reply contribute/create-pull-request.md:55]: I think @torkelo is working on a doc for the new theme. Will do that once it's published.

File: contribute/create-pull-request.md [hunk 3]
```diff
@@ -48,6 +48,13 @@ Pull requests that add or modify unit tests that are written in Jest must adhere
 - Don't add snapshots tests. We are incrementally removing existing snapshot tests, we don't want more.
 - If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), unless you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.
 
+Pull requests that create new UI components must adhere to the following accessibility guidelines:
+
+- Use semantic HTML.
+- Use aria roles, labels and other accessibility attributes correctly. Accessibility attributes should only be used when semantic HTML doesn't satisfy your use case.
```
Comment (marcusolsson) [inline contribute/create-pull-request.md:54]:
```suggestion
- Use ARIA roles, labels and other accessibility attributes correctly. Accessibility attributes should only be used when semantic HTML doesn't satisfy your use case.
```

I think ARIA is an acronym, right?

Comment (thisisobate) [reply contribute/create-pull-request.md:54]: yes, it is.
would update with some changes

**Review by marcusolsson — APPROVED** [review]

**Review by Elfo404 — APPROVED** [review]
Great start! 🚀
I only have some doubts regarding 
>Pull requests that create new UI components must adhere to the following accessibility guidelines:

as we should apply these rules not only when creating new components, but also when modifying, rewriting, move them around the page, and even deleting them. 

But regardless the phrasing this is a great start imho!

File: contribute/create-pull-request.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ Pull requests that add or modify unit tests that are written in Jest must adhere
 - Don't add snapshots tests. We are incrementally removing existing snapshot tests, we don't want more.
 - If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), unless you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.
 
-Pull requests that create new UI components must adhere to the following accessibility guidelines:
+Pull requests that create new UI components or modify existing ones must adhere to the following accessibility guidelines:
```
Comment (Elfo404) [inline contribute/create-pull-request.md:51]: ❤️

**Pull request merged**